CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ include_directories(SYSTEM ${CVC4_DIR} ${CVC4_DIR}/include)
 ## Boost ##
 ###########
 
+add_definitions(-DBOOST_NO_CXX98_FUNCTION_BASE)
```

Comment:
Fix described in https://github.com/boostorg/container_hash/issues/22

Reply:
Might be best to add this as a comment to the file itself. Otherwise this explanation is likely to get lost?

Reply:
Not sure this comment will be too relevant later on?

Reply:
I think if someone is coming in three years time and is wondering why this line exists and whether it's still needed, then it would be useful documentation?

Reply:
They could probably just look this flag up. :) But sure, doesn't hurt to add the comment.